extensions/wikia/Flags/controllers/FlagsController.class.php

```diff
@@ -45,21 +45,88 @@ private function processRequest() {
 	 */
 
 	/**
-	 * Retrieves all data for flags assigned to the given page.
+	 * Retrieves all data for all flag types available on the given wikia
+	 * with an intent of rendering a modal with an edit form.
+	 * It returns
+	 * @see getFlagsForPageForEdit()
```

Comment:
this is actually getFlagsForPageForEdit documentation, so this line is unnecessary?